autoload/leetcode.py

```diff
@@ -132,9 +161,12 @@ def is_login():
 def signin(username, password):
     global session
     session = requests.Session()
-    res = session.get(LC_LOGIN)
+    if LC_BASE.find("cn"):
+        res = session.get(LC_CSRF)
```

Comment:
Just for my curiosity, I would like to know if it's possible to acquire the initial CSRF from LC_CSRF even for leetcode.com so that we don't need this if-statement any more.

Reply:
In my attempt, there is no such thing in leetcode.com

Reply:
Thanks!